scio-test/src/test/scala/com/spotify/scio/avro/AvroUtils.scala

```diff
@@ -18,55 +18,31 @@
 package com.spotify.scio.avro
 
 import org.apache.avro.Schema
-import org.apache.avro.generic.{GenericData, GenericRecord}
+import org.apache.avro.generic.{GenericRecord, GenericRecordBuilder}
 
 import scala.jdk.CollectionConverters._
 
 object AvroUtils {
-  private def f(name: String, tpe: Schema.Type) =
-    new Schema.Field(
-      name,
-      Schema.createUnion(List(Schema.create(Schema.Type.NULL), Schema.create(tpe)).asJava),
-      null: String,
-      null: AnyRef
-    )
 
-  private def fArr(name: String, tpe: Schema.Type) =
-    new Schema.Field(name, Schema.createArray(Schema.create(tpe)), null: String, null: AnyRef)
-
-  val schema: Schema = Schema.createRecord("GenericTestRecord", null, null, false)
```

Comment:
there is no difference between a parsed schema and manually created. remove the code